<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tz60widget</class>
 <widget class="QWidget" name="tz60widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>-7</x>
         <y>-26</y>
         <width>1041</width>
         <height>631</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/Resources/sixties.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="hippie2_label">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>0</y>
         <width>321</width>
         <height>311</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/Resources/sixties.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/Resources/hippie.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="hippie1_label">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>0</y>
         <width>321</width>
         <height>311</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/Resources/sixties.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/Resources/hippie_talking.png</pixmap>
       </property>
      </widget>
      <widget class="QPushButton" name="startConvoButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>320</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">selection-background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));

background-color: rgb(0, 170, 255);</string>
       </property>
       <property name="text">
        <string>Start Conversation</string>
       </property>
      </widget>
      <widget class="QFrame" name="speechBubble">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>531</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-style: solid;
border-width:1px;
border-radius:50px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="speechBubbleText">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>411</width>
          <height>111</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QToolButton" name="nextButton">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>60</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border: none;
 max-width:30px;
 max-height:30px;
 min-width:30px;
 min-height:30px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QFrame" name="foodOptions">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>20</y>
          <width>121</width>
          <height>111</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="noFoodWarning">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>111</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 70, 38);
padding:3px;</string>
         </property>
         <property name="text">
          <string>Warning: 
No food in bag to 
give hippie</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="giveHippieFoodButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>101</width>
           <height>51</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(55, 235, 255);</string>
         </property>
         <property name="text">
          <string>Give hippie food
 from backpack</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="jointOptions">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>20</y>
          <width>121</width>
          <height>111</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="noJointWarning">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>111</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 70, 38);
padding:3px;</string>
         </property>
         <property name="text">
          <string>Warning: 
No joint in bag to 
give hippie</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="giveHippieJointButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>101</width>
           <height>51</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(55, 235, 255);</string>
         </property>
         <property name="text">
          <string>Give hippie joint
 from backpack</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QPushButton" name="tokenButton">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>200</y>
         <width>121</width>
         <height>121</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
background-image: url(:/Resources/token.png);
 border-width:1px;
 border-radius:50px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="buttonImage" stdset="0">
        <pixmap resource="resources.qrc">:/Resources/token.png</pixmap>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
